src/c++/perf_analyzer/report_writer.cc

```diff
@@ -110,6 +117,10 @@ ReportWriter::GenerateReport()
         ofs << ",Total GPU Memory";
       }
     }
+    if (should_output_llm_metrics_) {
+      ofs << ",Avg First Token Latency";
```

Comment:
I was wondering whether instead of llm specific terminology this could be something more generic. For example, "Avg First Response Latency" and "Response to Response Latency"?

Reply:
Good point. I actually thought about this and agree that it's more generic/accurate than using "token". However, at the same time, since the first token latency and token-to-token latency are pretty established terms in LLM space compared to "response latency", I wasn't sure if introducing a new term would be a good thing for the users (since they would have to translate Response -> Tokens). Since these metrics are going to be only valid for LLM models, maybe it's fine to go with "token latency". What do you think @tgerdesnv ?

Reply:
These are LLM metrics (and will only be printed for LLM models) so I'm ok with them being LLM-specific, unless Iman is claiming that our token metrics are inaccurate with this comment?
"I think we are probably never 100% sure about the response to token mapping from Triton level."

Reply:
I think he means it's up to the models/backends to determine how many tokens to include in each response for a streaming LLM

Reply:
Yes, there is another story to differentiate between LLM vs non-LLM case. See the comment above the implementation of should_output_llm_metrics(). Assuming that story can be correctly implemented then Hyunjae's changes here are fine. This is all going into a side branch for now.

Reply:
Thanks for the explanation. Resolving.